src/features/unified-share-modal/SharedLinkPermissionMenu.js

```diff
@@ -62,8 +70,11 @@ class SharedLinkPermissionMenu extends Component<Props> {
             trackingProps,
         } = this.props;
         const { sharedLinkPermissionsMenuButtonProps } = trackingProps;
-        const canShowTag = sharedLinkEditTagTargetingApi ? sharedLinkEditTagTargetingApi.canShow : false;
         const canShowTooltip = sharedLinkEditTooltipTargetingApi ? sharedLinkEditTooltipTargetingApi.canShow : false;
+        // we show the "new" tag if the tooltip is shown, otherwise we follow the normal targeting api flow
+        const canShowTag = sharedLinkEditTagTargetingApi
+            ? (isSharedLinkEditTooltipShown && !this.state.hasSeenEditTag) || sharedLinkEditTagTargetingApi.canShow
```

Comment:
It's usually prudent to keep ternaries to a single line and break out these compound conditions into their own variables for readability's sake.